src/components/Forms/create-repair-request.tsx

```diff
@@ -0,0 +1,169 @@
+// Page for submitting a repair request
+// This form is copied from /repair-request/index.tsx
```

Comment:
Good idea to move this into a separate component, but now there are two identical components to maintain. You should make this form reusable, and use it in both places.